tasks/fix-cat1.yml

```diff
@@ -408,9 +408,8 @@
       name:
           - tftp
           - tftp-server
-      state: absent
+      state: "{{rhel7stig_tftp_required | ternary('present', 'absent') }}"
```

Comment:
I don't think we should install anything here.

Reply:
I concur I don’t think the role should install these packages when the “required” variable is set to true. Should just remove when required = false. The package/service install is up to another role.